packages/playwright-test/src/reporters/base.ts

```diff
@@ -72,6 +73,10 @@ export class BaseReporter implements Reporter  {
     this.monotonicStartTime = monotonicTime();
     this.config = config;
     this.suite = suite;
+    this.totalTestCount = suite.allTests().length;
+    const jobs = Math.min(config.workers, (config as any).__testGroupsCount);
+    const shardDetails = config.shard ? `, shard ${config.shard.current} of ${config.shard.total}` : '';
+    console.log(`\nRunning ${this.totalTestCount} test${this.totalTestCount > 1 ? 's' : ''} using ${jobs} worker${jobs > 1 ? 's' : ''}${shardDetails}\n`);
```

Comment:
If this is now a terminal reporter, we should rename it to the terminal.ts and maybe extract all the utility functions that are used from non-terminal reporters into base.ts?